contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -142,6 +142,7 @@ static SimpleEcontextStackEntry *simple_econtext_stack = NULL;
 	MemoryContextAlloc(get_eval_mcontext(estate), sz)
 #define eval_mcontext_alloc0(estate, sz) \
 	MemoryContextAllocZero(get_eval_mcontext(estate), sz)
+char *original_query_string = NULL;
```

Comment:
* We should put this statement with all the other variable declaration above.
* Let's make this `static` if it is used only in this file.